### PR TITLE
Add Logan to organizers 

### PR DESCRIPTION
**Pull request by logankilpatrick**

### CODE REVIEW

File: _organizers/logan-kilpatrick.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+github: logankilpatrick
+layout: base
+name: Logan Kilpatrick
+photo_url: /static/img/organizers/logan.jpg
+role: DEFNA, DjangoCon 2022 Chair
```
Comment (drewbrew) [inline _organizers/logan-kilpatrick.md]:
```suggestion
role: DjangoCon US 2022 Chair, DEFNA Board Member
```

File: _organizers/logan-kilpatrick.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+twitter: OfficialLoganK
+website: [https://jefftriplett.com/](https://scholar.harvard.edu/logankilpatrick)
```
Comment (drewbrew) [inline _organizers/logan-kilpatrick.md]:
```suggestion
website: [https://scholar.harvard.edu/logankilpatrick](https://scholar.harvard.edu/logankilpatrick)
```

I know whose organizer file you copied! 😆

**Review by drewbrew — APPROVED** [review]

File: _organizers/logan-kilpatrick.md (hunk 2 above)
Comment (logankilpatrick) [reply _organizers/logan-kilpatrick.md]: Lol, whoops : )

**Comment by williln** [comment]
@logankilpatrick  Your website link needs to be just the link -- not formatted like a markdown link. The template takes care of that part. Thank you!

**Comment by logankilpatrick** [comment]
@williln Fixed, Thank you!

**Pull request merged**